subsys/bluetooth/controller/Kconfig

```diff
@@ -143,6 +143,7 @@ comment "BLE Controller configuration"
 config BT_CTLR_CRYPTO
 	bool "Enable crypto functions in Controller"
 	default y
+	select ENTROPY_GENERATOR
```

Comment:
The commit containing this change is tagged `[nrf noup]`. Shouldn't it actually go upstream? Enabling `BT_CTLR_CRYPTO` [causes](https://github.com/zephyrproject-rtos/zephyr/blob/master/subsys/bluetooth/controller/CMakeLists.txt#L17) `subsys/bluetooth/controller/crypto/crypto.c` to be compiled and this file [expects](https://github.com/zephyrproject-rtos/zephyr/blob/master/subsys/bluetooth/controller/crypto/crypto.c#L26) an entropy generator to be available.